infra/modules/cloudfront-certificate/main.tf

```diff
@@ -0,0 +1,33 @@
+provider "aws" {
+  region = "us-east-1"
```

Comment:
shouldn't this be our `eu-west` region

Reply:
No, the certificate for cloudfront distributions is always located in us-east-1.

Reply:
I see, I already supposed it would be something like that